CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
 if(POLICY CMP0092)
     cmake_policy(SET CMP0092 NEW)
 endif()
+if(POLICY CMP0114)
```

Comment:
@mikedld JFYI https://cmake.org/cmake/help/latest/policy/CMP0114.html

Reply:
Not sure why you're tagging me, but also not sure how this relates to Xcode and its recommended settings...

Reply:
@nevack this is still showing as an unresolved conversation. Is any decision / discussion still needed, or can this be resolved?

Reply:
Sorry, I wasn't aware we need to resolve this somehow.
TBH, I can't even find the control to mark this as resolved now